samples/HelloSignalR/wwwroot/index.html

```diff
@@ -0,0 +1,62 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <script src="https://code.jquery.com/jquery-2.1.4.js" type="text/javascript"></script>
+    <script src="Scripts/jquery.signalR-2.2.0.js" type="text/javascript"></script>
+    <link href="styles/sample.css" rel="stylesheet" />
+    <title>ASOS SignalR Sample</title>
+    <script type="text/javascript">
+        'use strict';
+        $(document).ready(function () {
+            let connectionBase = "http://localhost:5000";
+            let connectionPath = `${connectionBase}/signalr`;
+
+            let connectToSignalR = (token) => {
+                let connection = $.connection(connectionPath);
+                let log = data => console.log(data);
+
+                connection.qs = `access_token=${token}`;
+
+                connection.received(data => {
+                    log(`From server: ${data}`);
```

Comment:
That would be great if - instead of logging - you could just update the UI :smile: